model/cluster_installation_test.go

```diff
@@ -126,7 +126,7 @@ func TestClusterInstallationsFromReader(t *testing.T) {
 		]`)))
 		require.NoError(t, err)
 		require.Equal(t, []*ClusterInstallation{
-			&ClusterInstallation{
+			{
```

Comment:
hey cool I didn't know you could omit the type like this, that's nice

Reply:
Yeah, my linter started complaining about this stuff so I am fixing it when I see it.